torch/csrc/api/include/torch/nn/options/activation.h

```diff
@@ -315,5 +315,102 @@ struct TORCH_API GumbelSoftmaxFuncOptions {
 
 } // namespace functional
 
+// ============================================================================
+
+/// Options for MultiheadAttention functional and module.
+struct TORCH_API MultiheadAttentionOptions {
+  MultiheadAttentionOptions(int64_t embed_dim, int64_t num_heads)
+    : embed_dim_(embed_dim), num_heads_(num_heads) {
+    if (!kdim_) {
+      kdim_ = embed_dim;
+    }
+    if (!vdim_) {
+      vdim_ = embed_dim;
+    }
```

Comment:
I believe `kdim_` and `vdim_` are always equal to `c10::nullopt` when the constructor is run, and we can probably make the type of `kdim_` and `vdim_` non-optional, and always assign them the same value as `embed_dim` in the  constructor instead.